docs/configuration.md

```diff
@@ -3,4 +3,5 @@
 ### As a service:
 
 * I will take in environment variables natively to my service
-* I will not use configuration files where I can avoid it over using environment variables
+* Any environment specific configuration should be passed in as an environment variable
```

Comment:
- Any environment specific configuration should be passed in at runtime as environment variable(s) using a mechanism such as 'secrets'
- Any environment specific requirements e.g. certificates should be passed in to the environment at runtime using a mechanism such as 'secrets'